### DISCUSSION TITLE
Python 3.10 and 3.11 Support

### DISCUSSION THREAD

**Discussion by CristianoPizzamiglio** [pyvista/pyvistaqt#645] [Q&A] [answered]
Hi! Thank you so much for this great library!

Has the library been tested on Python 3.10 and Python 3.11? Thanks!

**Comment by tkoyama010** [comment, marked as answer]
Yes, could you please check the GitHub Actions CI file?
https://github.com/pyvista/pyvistaqt/blob/dc8c21aca54fb186b205c723150ca3e68cf9a68c/.github/workflows/ci.yml#L82-L112

**Comment by CristianoPizzamiglio** [comment]
@tkoyama010 Thank you so much!
